src/cdtp/page.rs

```diff
@@ -78,6 +78,21 @@ pub mod methods {
         type ReturnObject = CaptureScreenshotReturnObject;
     }
 
+    #[derive(Serialize, Debug)]
+    #[serde(rename_all = "camelCase")]
+    pub struct Reload<'a> {
+        pub ignore_cache: bool,
```

Comment:
This is an optional param at the protocol level. I'm not sure how important it is to have 1:1 parity, just thought I'd mention it.

Reply:
Yes, I saw that. Yet I'm hesitant to expose any optional parameter that has some default on the protocol level to the API, because every API-call looks like "here are a bunch of parameters and they are all their implicit default, go figure!" I'd strongly favor having less `Option`-type parameters on the public API.